Stripe/PublicHeaders/STPPaymentContext.h

```diff
@@ -233,6 +233,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, assign) UIModalPresentationStyle modalPresentationStyle;
 
+/**
+ The mode to use when displaying the title of the navigation bar in all view
+ controllers presented by the context. The default value is `automatic`,
+ which causes the title to use the same styling as the previously displayed
+ navigation item.
+ */
+@property (nonatomic, assign) UINavigationItemLargeTitleDisplayMode largeTitleDisplayMode;
```

Comment:
Might want to consider adding in some extra notes about how this property interacts with push vs present. Eg. the default of `automatic` won't work for present because there is no previous item. And for push we don't alter the owning navigation bar, you have to make sure it's navigation bar has `prefersLargeTitles = YES` in order for this to do anything.

Reply:
ah, good call – I'll expand